core/sequencer/server.go

```diff
@@ -92,7 +102,34 @@ func NewServer(
 		tlog:      tlog,
 		tmap:      tmap,
 		mutations: mutations,
+		queue:     queue,
+	}
+}
+
+// RunBatch reads mutations out of the queue and calls CreateEpoch.
+func (s *Server) RunBatch(ctx context.Context, in *spb.RunBatchRequest) (*empty.Empty, error) {
+	batch, err := s.queue.ReadQueue(ctx, in.DomainId, in.MaxBatch)
+	if err != nil {
+		return nil, status.Errorf(codes.Internal, "ReadQueue(): %v", err)
 	}
+	if int32(len(batch)) < in.MinBatch {
```

Comment:
If `in.MinBatch` is ever  > 1, can you end up with a single mutation queued up that never gets applied?

Reply:
Potentially - and that obviously isn't ideal.  
I was working on an assumption that x mutations were being added to the queue per second. 

A better approach might be to implement a timeout setting for sizes below the min batch size, but that's probably worth investigating in a separate PR. I'll add a note and a TODO.